README.md

```diff
@@ -3,15 +3,23 @@
 [![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)][godoc]
 [![Travis CI](https://travis-ci.org/secrethub/secrethub-go.svg?branch=master)][travis-ci]
 [![GolangCI](https://golangci.com/badges/github.com/secrethub/secrethub-go.svg)][golang-ci]
-[![Go Report Card](https://goreportcard.com/badge/github.com/secrethub/secrethub-go)][goreportcard]
+[![Go Report Card](https://goreportcard.com/badge/github.com/secrethub/secrethub-go)][goreportcard] ![Licence](https://img.shields.io/hexpm/l/plug.svg)
+
+
 
 The official [SecretHub][secrethub] Go client library.
 
-> SecretHub is a developer tool to help you keep database passwords, API tokens, and other secrets out of IT automation scripts. 
+> SecretHub is a developer tool to help you keep database passwords, API tokens, and other secrets out of IT automation scripts.
 
 <img src="https://secrethub.io/img/secrethub-gopher.png" alt="Gopher" width="200px"/>
 
-## Installation
+## Getting started
+
+### Prerequisites
+
+In order to use the Go client library, you need a __SecretHub account__. You can do this by following the first instructions [here](https://secrethub.io/docs/getting-started/) (*Installing the CLI* and *signing up for a SecretHub account*).
```

Comment:
What do you think of:

In order to use the SecretHub Go client library, you need an account on SecretHub. You can create a free account by [signing up through the CLI](https://secrethub.io/docs/getting-started/).

?

Reply:
Yes make it that.

Reply:
I agree on that.